forum/models.py

```diff
@@ -103,6 +103,12 @@ def get_author(self):
 	def get_response_count(self):
 		return Response.objects.filter(thread=self.id).count()
 
+	def get_all_authors(self):
+		authors = []
+		for r in self.response_set.all():
+			authors.append(r.author)
+		return list(set(authors))
```

Comment:
`return list(set([x.author for x in self.response_set.all()]))`?

Reply:
Or to be less brackety,
```
authors = [x.author for x in self.response_set.all()]
return list(set(authors))
```

Reply:
There might be a more Django-y way of doing it that'd translate to database joins too, but eh.

Reply:
I'll take the Pythonic way. :1234: